bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAdmin.java

```diff
@@ -360,15 +362,82 @@ public void deleteColumnFamily(TableName tableName, byte[] columnName) throws IO
   }
 
   @Override
-  public void modifyTable(TableDescriptor arg0) throws IOException {
-    throw new UnsupportedOperationException("modifyTable"); // TODO
+  public void modifyTable(TableDescriptor tableDescriptor) throws IOException {
+    modifyTable(tableDescriptor.getTableName(), tableDescriptor);
   }
 
   @Override
-  public void modifyTable(TableName arg0, TableDescriptor arg1) throws IOException {
-    throw new UnsupportedOperationException("modifyTable"); // TODO
+  public void modifyTable(TableName tableName, TableDescriptor tableDescriptor) throws IOException {
+    //Check table exists or not
+    if(isTableAvailable(tableName)) {
```

Comment:
Please get the current descriptor here.  Once you have that, we need to create an array of modifications that need to occur.  Those modifications need to be sent in a single RPC.

There's a method in AbstractBigtableAdmin:

```
protected void modifyColumn(TableName tableName, String columnName,
      String modificationType, Modification... modifications) 
```
Please use this method with an array of modifications.  That will send a single RPC.